src/Abp.RedisCache/Runtime/Caching/Redis/DefaultRedisCacheSerializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Abp.Json;
+using StackExchange.Redis;
+
+namespace Abp.Runtime.Caching.Redis
+{
+    /// <summary>
+    ///     Default implementation uses JSON as the underlying persistence mechanism.
+    /// </summary>
+    public class DefaultRedisCacheSerializer : IRedisCacheSerialization
```

Comment:
IRedisCacheSerialization should be named as IRedisCacheSerializer.
Also, DefaultRedisCacheSerializer should implement ITransientDependency (to register to DI).